Readme.md

```diff
@@ -71,8 +70,10 @@ const path = './Assets/Scripts/'
 
 gulp.task('build:js', () => jsTargets.compile(path, './wwwroot/directory-to-copy-into'));
 ```
-
 The base rules for the linter are found in the `eslint-base.js` file, you can add your own rules in the `.eslintrc` file overriding the base rules. 
+
+These base rules are from the [airbnb](https://github.com/airbnb/javascript) javascript style guide.
+
 These files are copied to the `src` directory on build, in order for them to work properly with ESLint.
```

Comment:
Ezután kellene plusz egy sortörés, mert `##` header jön.

Reply:
Nem feltétlen látom már mire gondolsz, mert előbb írtam bele a "how to update" részt és belekavart, de egységes a formázás a file többi részéhez most

Reply:
A lényeg, hogy `## header` előtt két sortörés legyen, ne egy.

Reply:
Okok